src/hooks/useRoleCapabilitySets.js

```diff
@@ -30,11 +33,21 @@ const useRoleCapabilitySets = (roleId) => {
     return obj;
   }, {});
 
+  const capabilitySetAppIds = useMemo(() => {
+    const capabilitySetsById = mapValues(keyBy(data?.capabilitySets, 'applicationId'), () => true) || {};
+    const filteredByInstalledApplications = pick(capabilitySetsById, installedApplications);
+
+    return filteredByInstalledApplications;
+    // stripes.discovery is configured during application initialization
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [data]);
+
   return { initialRoleCapabilitySetsSelectedMap,
     isSuccess,
     capabilitySetsTotalCount: data?.totalRecords || 0,
     groupedRoleCapabilitySetsByType,
-    capabilitySetsCapabilities };
+    capabilitySetsCapabilities,
+    capabilitySetAppIds };
```

Comment:
The parallel value in `useRoleCapabilities` is `capabilitiesAppIds`. Parallel structures should have parallel names. I think `capabilitySetsAppIds` is correct.

Reply:
You are right, fixed